### DISCUSSION TITLE
QRCode is not read by default IPhone Camera appliction

### DISCUSSION THREAD

**Discussion by sshousha** [Shane32/QRCoder#336] [General]
We print a QRCode in one of our SSRS reports , the barcode is not read by iPhone. it is working fine with Android. Note that the same iPhone can read other QRCodes (not generated from the DLL) by the default camera application. What  could be wrong.  Here is the code we use:

Public Function QRCoderByteArray(ByVal inputString As String) as Byte()
     Return QRCoder.BitmapByteQRCodeHelper.GetQRCode(inputString, QRCoder.QRCodeGenerator.ECCLevel.Q, 20)
End Function

**Comment by codebude** [comment]
This is the result of too much image compression/resizing of the QR code. See comments on #337 for explanation.
